packages/web-components/package.json

```diff
@@ -199,8 +198,7 @@
   "devDependencies": {
     "@types/web": "^0.0.142",
     "@storybook/html": "6.5.15",
-    "@tensile-perf/web-components": "~0.1.13",
-    "rimraf": "^3.0.2"
```

Comment:
node js can do this natively without abstraction that contains security vulnerabilities time to time